source/_integrations/iotmeter.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: IoTMeter
+description: Instructions on how to integrate IoTMeter within Home Assistant.
+ha_category:
+  - Energy
+ha_release: 2024.6
+ha_iot_class: Cloud Polling
+ha_domain: iotmeter
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_codeowners:
+  - '@lipic'
+ha_integration_type: integration
+---
+
+The **IoTMeter** {% term integration %} allows you to monitor and manage your [IoT meters](http://evracing.cz/iotmeter/iotmeter_EN.pdf) within Home Assistant. This integration provides real-time data and control over your connected devices.
+
+{% include integrations/config_flow.md %}
+
+## Sensors
+
+The {% term integration %} adds the following sensors:
+
+- Total power (unit: kW)
+  This sensor measures the total power being consumed at any given time. It is useful for monitoring overall power usage and identifying peak consumption periods.
+
+- Total consumption energy (unit: kWh)
```

Comment:
```suggestion
- **Total energy consumption** (unit: kWh): 
```
I took the liberty to rephrase this UI string, as it seems more logical. However, it is important that the UI string is the same as the one shown in the software. So please feel free to ignore the rephrase if this does not work with the software.